examples/menubar/menubar-editor.html

```diff
@@ -471,7 +464,14 @@ <h3 id="rps1_label">Menubar</h3>
             <td>
               <code>span</code>
             </td>
-            <td>Indicates the menu is open.</td>
+            <td>
+              <ul>
+                <li>Indicates the submenu is open.</li>
+                <li>
+                  The visual appearance of the expanded state is synchronized with the <code>aria-expanded</code> value using CSS attribute selectors and hidden from screen readers with <code>aria-hidden="true"</code>.
```

Comment:
@jongund I see here that @zcorpan describes the use of `aria-hidden` on a span within the `menuitem` element. And the same has been done for `menuitemcheckbox` and `menuitemradio`. Just checking if you took note of this or is it that you would rather it be in a row for itself?

Reply:
`aria-hidden` needs it's own table row in the table.
Example code:

```
<tr data-test-id="menu-aria-hidden">
    <td></td>
    <th scope="row"><code>aria-hidden="true"</code></th>
    <td><code>span</code></td>
    <td>Removes the character entities used for icons from the accessibility tree to prevent them from being included in the accessible name of the sort buttons.</td>
</tr>
```